homework/test/test-list.js

```diff
@@ -1,8 +1,14 @@
-describe('Test > ', function() {
+describe('Test_sin > ', function() {
+  var subject;
+
   beforeEach(function() {
+    subject = new TestManager;
   });
 
-  it('will test some pure functions', function() {
+  it('will test the sin function', function(done) {
     // Write any pure function assertion here.
+	subject.set(0);
+    assert.Equal(subject.sin(), 0);
```

Comment:
**WRONG**: why did you think you can test a manager that **apparently doesn't exist in your patch**? And apparently, you're testing a method **belongs to the built-in Math object** which never is one method of the inexistent manager. Again, it would never happen if you test and write your code carefully. Please correct it.